transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Documents/UserInfo.java

```diff
@@ -0,0 +1,44 @@
+package com.transcriptanalyzer.transcript.User_Requests_Intents.Documents;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document
+public class UserInfo {
```

Comment:
As we have this pseudo superclass of API & Account, the actual individual API & Account classes are probably unnecessary and just add complexity. All the methods mentioned for UserInfo seem to cover what's mentioned in each of the subclasses - would likely reduce the number of files and improve readability and tracking!